src/main/java/com/dataloom/requests/PermissionsRequestDetails.java

```diff
@@ -0,0 +1,67 @@
+package com.dataloom.requests;
+
+import java.io.Serializable;
+import java.util.EnumSet;
+import java.util.Map;
+import java.util.UUID;
+
+import com.dataloom.authorization.Permission;
+import com.dataloom.data.SerializationConstants;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class PermissionsRequestDetails implements Serializable {
+    private static final long              serialVersionUID = -3977504707164513197L;
+
+    @JsonProperty( SerializationConstants.PERMISSIONS )
```

Comment:
I wouldn't expect that to pass - and oops I forgot to move this one to the getters. Will fix.